specs/capella/beacon-chain.md

```diff
@@ -115,6 +118,18 @@ class SignedBLSToExecutionChange(Container):
     signature: BLSSignature
 ```
 
+#### `HistoricalSummary`
+
+```python
+class HistoricalSummary(Container):
+    """
+    `HistoricalSummary` matches the components of the phase0 `HistoricalBatch`
+    making the two hash_tree_root-compatible.
+    """
+    block_batch_root: Root
+    state_batch_root: Root
```

Comment:
@djrtwo @arnetheduck 
Should we also rename `block_batch_root` to `block_summary_root` and `state_batch_root` to `state_summary_root`? The docstring hints it has a connection to `HistoricalBatch`, so I didn't change it, but I'm okay with either way.

Reply:
agree on changing the name to block/state_summary_root. although don't feel too strongly

Reply:
no objections :)